C/Cpp_include/c4Query.hh

```diff
@@ -117,21 +117,37 @@ protected:
 
 private:
     class LiveQuerierDelegate;
+
+    struct KeyCmp {
+        using is_transparent = void;
```

Comment:
What does this line do?

Reply:
This is to allow comparison to be done between Key type and another type. Otherwise, plain pointer will be converted to RefCounted before comparison.

Reply:
I mean specifically is_transparent = void.  I know what the class is for.

Reply:
Does matter what type is_transparent is aliased to, as long as the alias exists. c++ template quirky thing. std::set::find requires it. Without "is_transparent", "find" can only find by value of KeyType.